static/js/controllers/associations.controller.js

```diff
@@ -24,20 +20,14 @@ exports.getFromTables = (req, res) => {
             associations = data[0]
             traits = data[1]
 
-            returnData = await separateStudies(associations, traits, refGen, defaultSex)
+            returnData = await separateStudies(associations, traits, refGen)
```

Comment:
Is defaultSex here the sex requested by the user? If so, I believe we will still need it so the user can specify what studies they want to use.

Reply:
They specify earlier. we don't need to separate the studies using that variable because that is already built in to the pValueAnnotation